### PR TITLE
OI-2593 contribute view shows a secondary navigation with options and hides primary items with icon and selected state styling etc

### PR DESCRIPTION
**Pull request by moz-rotimib**
## NOTE
This PR should ONLY be merged and deployed after #3932 

- Adds secondary nav menu bar on Contribute page
- Adds menu icon for hiding and revealing primary navigation items
- Mobile states

### CODE REVIEW

File: web/locales/en/messages.ftl [hunk 1]
```diff
@@ -960,6 +961,9 @@ target-segment-first-banner = Help create Common Voice’s first target segment
 target-segment-add-voice = Add Your Voice
 target-segment-learn-more = Learn More
 
+## Contribution Nav Items
+contribute-nav-header-item-1=Voice Collection
```
Comment (moz-dfeller) [inline web/locales/en/messages.ftl:965]: Wouldn't a meaningful name make more sense here than a generic `item-1`?

```suggestion
contribute-voice-collection-nav-header = Voice Collection
```

File: web/src/components/layout/nav/contribute-menu-content.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+export const ContributeMenuContent: React.FC<ContributeMenuContentProps> = ({
+  className,
+  pathname = '',
+}) => {
+  const speakActive = pathname.includes(URLS.SPEAK);
+  const listenActive = pathname.includes(URLS.LISTEN);
+
+  return (
+    <div className={className}>
+      <Localized id="contribute-nav-header-item-1">
```
Comment (moz-dfeller) [inline web/src/components/layout/nav/contribute-menu-content.tsx:23]:
```suggestion
      <Localized id="contribute-voice-collection-nav-header">
```

**Review by moz-dfeller — COMMENTED** [review]
Just thinking that the l10n id could have a better naming. Else it looks good to me.

**Comment by moz-rotimib** [comment]
> Just thinking that the l10n id could have a better naming. Else it looks good to me.

Yeah that's a good idea. I'll fix that

**Pull request merged**